<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo List</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Current</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="currentListWidget"/>
        </item>
        <item row="13" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Move to Future</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Complete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;h3&gt;New Todo&lt;/h3&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Text:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Future</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListWidget" name="futureListWidget"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Complete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Completed</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListWidget" name="completedListWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>296</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>currentListWidget</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>futureListWidget</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>completedListWidget</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>861</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>moveToFutureButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>completeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>completeButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>831</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeButtonClick()</slot>
  <slot>addButtonClick()</slot>
  <slot>moveToFutureButtonClick()</slot>
  <slot>completeButtonClick()</slot>
  <slot>updateInterface()</slot>
 </slots>
</ui>
